scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -92,13 +92,13 @@ def main():
         return_error,
     )
     return_error = check_json(
-        output["context"]["encoding"]["default"]["encoding"],
+        output["context"]["encoding"]["preferred"]["encoding"],
         arguments["--encoding"].replace("'", ""),
         "Encoding doesn't match with JSON:",
         return_error,
     )
     return_error = check_json(
-        output["context"]["encoding"]["default"]["compression"],
+        output["context"]["encoding"]["preferred"]["compression"],
```

Comment:
Not here, but cannot set the mark elsewhere ...

Can you please adjust one of the cases to use preffered=='FoR' so we can check that a helpful message is printed?

Reply:
Exactly, can you expect something like that line here?
` - Column 'movie_info.note' of type string cannot be encoded as FrameOfReference and its encoding is chosen automatically.
1